client/src/components/common/applicant-info-2.vue

```diff
@@ -268,30 +265,29 @@ export default class ApplicantInfo2 extends Vue {
   set priorityRequest (value) {
     newReqModule.mutatePriorityRequest(value)
   }
-  set submissionTabNumber (value) {
-    newReqModule.mutateSubmissionTabNumber(value)
-  }
 
-  async submit () {
-    if (this.editMode) {
-      await newReqModule.patchNameRequests()
-    } else {
-      const { nrId } = this
-      if (!nrId) {
-        await newReqModule.postNameRequests('draft')
-      } else {
-        if (!this.editMode && ['COND-RESERVE', 'RESERVED'].includes(this.nrState)) {
-          const request = await newReqModule.getNameRequest(nrId)
-          if (request?.stateCd === 'CANCELLED') {
-            newReqModule.setActiveComponent('Timeout')
-            return
-          }
-        }
-        await newReqModule.putNameReservation(nrId)
-      }
-      await paymentModule.togglePaymentModal(true)
-    }
-  }
+  // NOT USED
```

Comment:
Commented out "decoy" code. Did not delete in case we need to reinstate it in the future.